index.rst

```diff
@@ -46,14 +46,314 @@
+   | CSC          | lsst-l1-cl-tss34 | 141.142.238.133 | - Camera Hexapod                      |
+   +--------------+------------------+-----------------+---------------------------------------+
+   | CSC          | lsst-l1-cl-tss35 | 141.142.238.134 | - M2 Hexapod                          |
+   +--------------+------------------+-----------------+---------------------------------------+
+   | CSC          | lsst-l1-cl-tss36 | 141.142.238.135 | - MTAOS                               |
+   +--------------+------------------+-----------------+---------------------------------------+
+   | CSC          | lsst-l1-cl-tss37 | 141.142.238.136 | - M1M3                                |
+   +--------------+------------------+-----------------+---------------------------------------+
+   | CSC          | lsst-l1-cl-tss38 | 141.142.238.137 | - M2                                  |
+   +--------------+------------------+-----------------+---------------------------------------+
+
+
+Although not all components subscribes to each other topics, the :ref:`network topology diagram <fig-net-topology-full-community>` displays an extremely high inter-connectivity between all the different components/hosts.
+To begin with, note the IP address ``239.255.0.1`` in the diagram, which is the multicast server and receives traffic from all the components.
+A more close look into the diagram show that there are two distinct behavior, related to the two different types of services deployed; CSCs and salkafka producers.
+
+CSCs will send and receive DDS traffic to all other components of the system, whereas salkafka producers will send and receive traffic from CSCs only.
+This is more easily seen by filtering all the inbound and outbound traffic to one of the CSCs and to one of the salkafka producers individually.
+Diagrams with the traffic to the MT pointing component (MTPtg, hostname ``lsst-l1-cl-tss31``) and to the salkafka producer for the AT components (hostname ``lsst-l1-cl-tss16``) are shown in the left and right hand side of :numref:`fig-net-topology-inset-community`, respectively.
+
+.. figure:: /_static/net_topology_inset_community.png
+   :name: fig-net-topology-inset-community
+   :target: ../_images/net_topology_inset_community.png
+   :alt: filtered network topology to the MT Rotator CSC and to the M1M3 producer
+
+   Filtered network connectivity to the MTPtg CSC (left hand side, hostname ``lsst-l1-cl-tss31``) and to the salkafka producer for AT components (right hand side, hostname ``lsst-l1-cl-tss16``).
+
+One of the reasons we selected the salkafka producer (``lsst-l1-cl-tss16``) for the diagram in :numref:`fig-net-topology-inset-community`, is that it does not subscribe to any of the CSCs running on the system at that time.
+Still, the component is receiving DDS traffic for all the CSCs.
+
+At the same time, the MTPtg is a component that no other CSC subscribe to events or telemetry from.
```

Comment:
... that surely won't be true in the future...  so maybe say "At the same time, the MTPtg is a component that no other CSC currently subscribes to the event or telemetry streams.